frontend/src/scenes/authentication/InviteSignup.tsx

```diff
@@ -151,226 +127,171 @@ function AuthenticatedAcceptInvite({ invite }: { invite: PrevalidatedInvite }):
-                                disabled={acceptedInviteLoading}
+                                loading={acceptedInviteLoading}
                             >
                                 Accept invite
-                            </Button>
+                            </LemonButton>
                             <div className="mt-4">
-                                <Link to="/">
-                                    <ArrowLeftOutlined /> Go back to PostHog
-                                </Link>
+                                <LemonButton type="secondary" center fullWidth icon={<IconChevronLeft />} to="/">
+                                    Go back to PostHog
+                                </LemonButton>
                             </div>
                         </>
                     ) : (
-                        <Button block onClick={() => (window.location.href = '/')}>
-                            Go to PostHog <ArrowRightOutlined />
-                        </Button>
+                        <LemonButton
+                            type="secondary"
+                            center
+                            fullWidth
+                            sideIcon={<IconChevronRight />}
+                            onClick={() => (window.location.href = '/')}
+                        >
+                            Go to PostHog
+                        </LemonButton>
                     )}
                 </div>
-            </Space>
-        </div>
+            </div>
+        </BridgePage>
     )
 }
 
 function UnauthenticatedAcceptInvite({ invite }: { invite: PrevalidatedInvite }): JSX.Element {
-    const [formValues, setFormValues] = useState({
-        firstName: invite?.first_name || '',
-        password: '',
-        emailOptIn: true,
-    })
-    const [formState, setFormState] = useState({ submitted: false, passwordInvalid: false })
-    const passwordInputRef = useRef<Input | null>(null)
-    const { acceptInvite } = useActions(inviteSignupLogic)
-    const { acceptedInviteLoading } = useValues(inviteSignupLogic)
-    const { socialAuthAvailable } = useValues(preflightLogic)
-
-    const parentContainerRef = useRef<HTMLDivElement | null>(null) // Used for scrolling on mobile
-    const mainContainerRef = useRef<HTMLDivElement | null>(null) // Used for scrolling on mobile
+    const { signup, isSignupSubmitting } = useValues(inviteSignupLogic)
+    const { preflight } = useValues(preflightLogic)
 
-    const goToMainContent = (): void => {
-        const yPos = mainContainerRef.current ? mainContainerRef.current.getBoundingClientRect().top : null
-        if (yPos) {
-            parentContainerRef.current?.scrollTo(0, yPos)
-        }
-    }
+    return (
+        <BridgePage
+            view={'invites-signup'}
+            message={
+                <>
+                    Welcome to
+                    <br /> PostHog{preflight?.cloud ? ' Cloud' : ''}!!!!
```

Comment:
That's a lot of exclamation marks!!!!

Reply:
I know right :D It's in the design but I'm not sure if we should stick with it. Feels a bit panicky 😨

Reply:
Making an executive decision and reducing it to just "!" :D